testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/WebViewActivity.kt

```diff
@@ -0,0 +1,49 @@
+package com.rakuten.tech.mobile.testapp.ui.display
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import com.rakuten.tech.mobile.testapp.ui.base.BaseActivity
+import com.rakuten.tech.mobile.testapp.ui.component.SampleWebView
+import com.rakuten.tech.mobile.testapp.ui.component.SampleWebViewClient
+
+class WebViewActivity: BaseActivity() {
+    private lateinit var sampleWebView: SampleWebView
+
+    companion object {
+        val loadUrlTag = "load_url_tag"
+        val miniAppUrlSchemesTag = "miniapp_url_schemes_tag"
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        showBackIcon()
+        title = "External WebView Sample"
+        val finishCallBack: (url: String) -> Unit = {
```

Comment:
Could we keep this functionality contained in the SDK if we require that they provide the Activity context?